modules/introduction/pages/intro.adoc

```diff
@@ -1,3 +1,87 @@
+
+The page at https://platformengineering.org/blog/what-is-platform-engineering created by the platform engineering community explains the details.
+
+== Why do you need an Internal Developer Platform (IDP)?
+
+The page at https://internaldeveloperplatform.org/why-build-an-internal-developer-platform provides a detailed explanation for why you need an IDP.
+
+Additionally, below is a short 6 minute introduction to Backstage and the rationale for building an Internal Developer Platform (IDP).
+
+video::n1IrNe5MmZg[youtube,title=Backstage.io - An Open Source Portal for Taming Developer Chaos,width=640,height=480]
+
+== Backstage, Project Janus and Red Hat Developer Hub
+
+image::backstage-janus-rhdh-flow.png[width=800px]
+
+=== Backstage
+
+**Backstage** is an open-source platform for building developer portals created by Spotify and donated to the Cloud Native Computing Foundation (CNCF). Rather than a product that is ready to use by developers, Backstage provides the core features for building developer portals, and has an extensive list of plugins supporting integration with platforms, runtimes and external APIs. Platform engineering teams take the base Backstage platform, and then extend and customize it for their developers' needs. Refer to the Backtage website for a full list of plugins.
+
+=== Project Janus
+
+**Project Janus** is a Red Hat-sponsored open source community for building IDPs and plugins using Backstage.
+It is the “mid-stream” for the Red Hat Developer Hub product, sitting between Backstage upstream and the supported Red Hat product offering which is focused on four areas:
+
+* Installation on OpenShift Container Platform
+* Developing and enhancing plugins, specifically targeting other Red Hat products like Red Hat OpenShift Pipelines, Red Hat OpenShift GitOps, Red Hat Single Sign-On (RHSSO) and many more
+* Providing "golden path" template examples and software templates
+* Improving the upstream Backstage platform
+
+Below is an hour long demonstration video and discussion about Janus IDP.
+
+video::Ck7r5DpEwj0[youtube,title=GitOps Guide to the Galaxy | Episode 60 - What's Janus IDP?,width=640,height=480]
+
+=== Red Hat Developer Hub
+
+**Red Hat Developer Hub** is the enterprise product that Red Hat builds and supports in production. It is based on the Janus project, and provides an integrated developer platform for customers. Several important plugins that integrate with other Red Hat Enterprise products are bundled in the product. The product is built for deployment on OpenShift Container Platform, and Red Hat builds, integrates and tests all the container images on Red Hat Enterprise Linux (RHEL) Universal Base Images (UBI). These UBI based container images are built for improved security and stability, and are constantly tested and updated to address security vulnerabilities.
```

Comment:
This summary mentions Janus (midstream) but doesn't refer to Backstage (upstream) any more? Is this intentionally, because e.g. the architecture diagram does the opposite -> has Backstage in there but is lacking the mention of Janus. It might get confusing at some point to new comers?

Reply:
Re-emphasizing again. The previous para also clearly mentions that Backstage is upstream and Janus is mid-stream.